launch.go

```diff
@@ -88,6 +93,23 @@ func createWorkspace(srcPaths ...string) (Workspace, error) {
 	return w, nil
 }
 
+// checkPR checks to see if the job name is a pull request and returns its number
```

Comment:
Hmm seems so, don't think SCM URL has an attribute indicating whether one of its jobs is a PR.